src/objects/Sprite.js

```diff
@@ -138,6 +165,7 @@ Sprite.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 				distance: distance,
 				point: intersectPoint.clone(),
+				uv: uvIntersection( intersectPoint, vA, vB, vC, uvA, uvB, uvC ),
```

Comment:
```javascript
uvIntersection( intersectPoint, vA, vB, vC, uvA, uvB, uvC ).clone();
```

Reply:
this will make sense if uvIntersection moved some place where it's public, and then usedin both sprite and mesh

Reply:
I think it is OK if it is fixed in both places.